install_config/upgrading/automated_upgrades.adoc

```diff
@@ -202,61 +221,155 @@ on what features are enabled.
 You can run the automated upgrade playbook using Ansible directly, similar
 to the advanced installation method, if you have an inventory file.
 
-The same *_v3_3_* upgrade playbook can be used to upgrade either of the
-following to the latest 3.3 release:
+The same *_v3_4_* upgrade playbooks can be used for either of the following
+scenarios:
+
+- Upgrading existing {product-title} 3.3 clusters to 3.4
+- Upgrading existing {product-title} 3.4 clusters to the latest
+xref:../../release_notes/ocp_3_4_release_notes.html#ocp-34-asynchronous-errata-updates[asynchronous
+errata updates]
+
+[[upgrading-control-plane-nodes-separate-phases]]
+=== Upgrading the Control Plane and Nodes in Separate Phases
+
+An {product-title} cluster can be upgraded in one or multiple phases. You can
+choose whether to upgrade all hosts in one phase with a single Ansible run, or
+upgrade the _control plane_ (master components and etcd) and nodes in separate
+phases using separate playbooks.
+
+When upgrading in multiple phases, the control plane upgrade phase includes:
+
+- master components
+- Docker only on any stand-alone etcd hosts
+
+It does does not include:
+
+- node services running on masters
+- Docker running on masters
+- node services running on stand-alone nodes
```

Comment:
I have a pending TODO to bring this doc to your attention once it's published: https://docs.google.com/a/redhat.com/document/d/1Jv5ROqosiG2-WhdWEkwpXPmvKbRaUC5yiLEAnZnzkeg/edit?usp=sharing

The point was raised that we don't clearly state if it's possible to do a zero downtime upgrade or not, and if so what would be required in terms of infrastructure. I don't know if there's time now to look into this but hopefully we can get something into official docs stating that you can do zero downtime upgrades if your app is capable, and you have sufficient replication for your app, extra nodes, and at least 3 infra nodes.

Also the description of the steps of upgrade might be useful.